examples/import/foo.arrai

```diff
@@ -0,0 +1 @@
+//{/examples/import/bar}
```

Comment:
Nit: maybe name these after the imports, e.g. `module_import.arrai` and `relative_import.arrai` or wahtever they actually are instead of `bar` and `bar2`.

Reply:
Also maybe add a short README in the exples/import dir to explain what this example is demonstrating and how to run it. In particular, that it works (assuming it does) whether you `arrai run bar.arrai` or `arrai run examples/import/bar.arrai`.